multitask/multitask_tutorial.py

```diff
@@ -194,14 +210,17 @@ def split_data(data, labels, seed=123):
     name="circle_task",
     module_pool=module_pool,
     task_flow=task_flow,
-    loss_func=partial(ce_loss, "circle_head"),
-    output_func=partial(softmax, "circle_head"),
+    loss_func=partial(ce_loss, op_name="circle_head"),
+    output_func=partial(softmax, op_name="circle_head"),
     scorer=Scorer(metrics=["accuracy"]),
 )
 
 # %% [markdown]
 # Note that `Task` objects are not dependent on a particular dataset; multiple datasets can be passed through the same modules for pre-training or co-training.
 
+# %% [markdown]
+# ### Again, but faster
```

Comment:
Do we really need to do this big simplicity reveal? I say we just do this version, then point people to the docs or superglue for the more advanced task spec